csrc/scheduler/matmul_heuristic.h

```diff
@@ -90,6 +90,11 @@ class MatmulParams : public HeuristicParams {
   //!    C3 C4 D3 D4
   int grid_swizzle_factor = 1;
 
+  //! use multiple thread blocks to calculate one tile
+  int split_k_factor = 1;
+  //! use multiple warps to calculate one sub-tile
+  //! slice_k_factor = cta_tile.k / warp_tile.k
```

Comment:
This functionality (`slice_k_factor`) will be added later?

Reply:
yes. let me remove it from this PR.